widget/embedded/src/constants/errors.ts

```diff
@@ -21,10 +21,10 @@ export const errorMessages = () => {
     },
     bridgeLimitErrors: {
       increaseAmount: i18n.t(
-        'Bridge Limit Error: Please increase your amount.'
+        'Bridge Limit Error: Please increase your amount to at least'
       ),
       decreaseAmount: i18n.t(
-        'Bridge Limit Error: Please decrease your amount.'
+        'Bridge Limit Error: Please decrease your amount to at least'
```

Comment:
Please increase your amount to at least 10,000 BUSD.
Please reduce your amount to less than 10,000 BUSD.